app/helpers/dashboard_helper.rb

```diff
@@ -43,6 +43,12 @@ def format_period(period, value)
     period == :month ? value.strftime("%b-%Y") : quarter_string(value)
   end
 
+  def multiline_format_period(period, value)
+    formatted_period = format_period(period, value)
+    shortened_period, year = formatted_period.split("-")
+    shortened_period + "-\n" + year
```

Comment:
The newline here doesn't introduce a newline in the markup, right? The newline would be interpreted as a space in the markup, and the word-wrap behavior would kick in? Shall we just introduce a space here?

Consider something like `format_period(period, value).gsub("-", "- ")`

Reply:
This is a good suggestion, thank you. I knew something felt off about using split but I didn't think of gsub. You're probably also right about the newline. I'm not sure how the view would have interpreted that. But I think you're correct that it's unnecessary in this case. This table is already cramped for space so I don't think it would ever appear on a single line.

Reply:
After talking to Claudio, we decided to use the carriage return after all because if we just add a space and the table has space for the full date, it would appear as "MON- YEAR" which seems worse than:
MON-
YEAR